src/Controls/samples/Controls.Sample.UITests/Issues/Issues15815.xaml

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issues15815"
+             Title="Issues15815">
+    <Grid RowDefinitions="Auto,Auto, Auto" >
+        <Label Grid.Row="0" Text="The last cell (index 2) is not visible with MeasureAllItems if 1st cell is 50px width" />
```

Comment:
Could we cover with tests other conditions that could impact the size?
- Vertical List
- Grouping

Maybe a root page with 3 buttons navigating to specific samples or a Grid with rows and different CollectionViews.

Reply:
hum right now I wanted to get this fixed and with a test. I think we can add more test pages in the future.

Reply:
Added an issue: https://github.com/dotnet/maui/issues/20465